ray-operator/controllers/ray/common/pod.go

```diff
@@ -308,54 +331,59 @@ func labelPod(rayNodeType rayiov1alpha1.RayNodeType, rayClusterName string, grou
 	return labels
 }
 
-func setInitContainerEnvVars(container *v1.Container, svcName string) {
+func setContainerEnvVar(container *v1.Container, key string, value string) {
```

Comment:
Container env-setting logic is currently repeated several times. Factored it out into a function.